### DISCUSSION TITLE
Welcome to general-alarm-device Discussions!

### DISCUSSION THREAD

**Discussion by LokiMetaSmith** [PubInv/general-alarm-device#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LokiMetaSmith** [comment]
Hi, Looks like this might be a good place to start with general project related discussions.

**Comment by RobertLRead** [comment]
Here is a general architecture diagram that perhaps can be a useful touchstone for discussion:


![GPAD Architecture](https://user-images.githubusercontent.com/5296671/174829779-f512d886-b4b6-4605-b370-dfc2ff5c4850.png)
